test/prototype_common_utils.py

```diff
@@ -527,3 +529,42 @@ def make_mask_loaders(
 
 
 make_masks = from_loaders(make_mask_loaders)
+
+
+class VideoLoader(ImageLoader):
+    pass
+
+
+def make_video_loader(
+    size="random",
+    *,
+    color_space=features.ColorSpace.RGB,
+    num_frames="random",
+    extra_dims=(),
+    dtype=torch.float32,
+):
+    size = _parse_image_size(size)
+    num_frames = int(torch.randint(1, 6, ())) if num_frames == "random" else num_frames
+
+    def fn(shape, dtype, device):
+        video = make_image(size=shape[-2:], color_space=color_space, extra_dims=shape[:-2], dtype=dtype, device=device)
+        return features.Video(video, color_space=color_space)
+
+    return VideoLoader(fn, shape=(*extra_dims, num_frames, *size), dtype=dtype, color_space=color_space)
+
+
+def make_video_loaders(
+    *,
+    sizes=DEFAULT_IMAGE_SIZES,
+    color_spaces=(
+        features.ColorSpace.GRAY,
+        features.ColorSpace.RGB,
+    ),
+    num_frames=(1, 0, "random"),
+    extra_dims=DEFAULT_EXTRA_DIMS,
+    dtypes=(torch.float32, torch.uint8),
```

Comment:
For now we only support uint8, what are the implications of having float32 in this tuple?

Reply:
It means all our transforms test that don't explicitly specify the `dtype` will test with videos in both `dtypes`. If we are only supporting `torch.uint8` now, we should remove this here to avoid unnecessary tests that might or might not pass.

Reply:
```suggestion
    dtypes=(torch.uint8,),
```